docs/controls/RemoteDevicePicker.md

```diff
@@ -0,0 +1,74 @@
+---
+title: RemoteDevicePicker Control
+author: avknaidu
+description: The RemoteDevicePicker control helps you to pick a device that you can use to Remote Launch Apps, Services.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, RemoteDevicePicker, picker
+dev_langs:
+  - csharp
+---
+
+# RemoteDevicePicker Control 
+
+The [RemoteDevicePicker](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.controls.remotedevicepicker) gives you a list of Remote Systems. All the systems must be signed in with the same Microsoft Account (MSA)
+
+> [!IMPORTANT]
+> Make sure you enable "RemoteSystem" Capability in `package.appxmanifest`
```

Comment:
Can we keep the capability doc link here to be consistent with our CameraPreview format for this message?
```suggestion
> Make sure you enable the [RemoteSystem capability](https://docs.microsoft.com/en-us/windows/uwp/packaging/app-capability-declarations#general-use-capabilities) in your app's `package.appxmanifest` to access remote system information.
```